### DISCUSSION TITLE
sap_storage_setup

### DISCUSSION THREAD

**Discussion by woreis** [sap-linuxlab/community.sap_install#750] [Q&A]
Hello all,
I was testing the sap_storage_setup role. It worked fine for me but I was missing the possibility to pass on special options for volume creation like --dataalignment or blocksize for filesystem creation.
And as far as I understood the role, when using striping I can only stipe over all disks. I'm missing something like striping over 4 disks when I have in total 8 disks available.
Are you planning to enhance the role regarding this ?

Cheers, Wolfgang

**Comment by sean-freeman** [comment]
Hi Wolfgang, the Ansible Role received a major overhaul in early 2024 to provide more SAP intrinsic functionality. The operators like `--dataalignment` are still there, please see the var file which shows this:
https://github.com/sap-linuxlab/community.sap_install/blob/main/roles/sap_storage_setup/defaults/main.yml#L24-L35
